.github/workflows/qhelp-pr-preview.yml

```diff
@@ -0,0 +1,39 @@
+name: Query help preview
```

Comment:
Don't we already have a job that does this for all languages?

Reply:
There is no workflow for that in this repository. There is another indirect check that verifies qhelp files, however, the logs of that job are not available to open source developers. In addition this job posts the markdown previews as a comment in the pull request for easy reviewing.